docs/vsphere_ci/scripts/autounattend.xml

```diff
@@ -101,16 +101,21 @@
                 </SynchronousCommand>
                 <SynchronousCommand wcm:action="add">
                     <Order>4</Order>
+                    <Description>Configure VMware Tools</Description>
```

Comment:
Don't you need to add `<ComputerName>` somewhere in this file? Following the instructions from this PR is resulting in VMs that have hostname set in CAPS which causes the kubelet configuration to fail.

Reply:
Good catch, thank you. Looks like missed it in the rebase process.